### PR TITLE
upgraded to TF 0.14 and fix zip file issue

### PR DESCRIPTION
**Pull request by SadriG91**

### CODE REVIEW

File: modules/fargate/fargate/main.tf [hunk 1]
```diff
@@ -169,10 +169,18 @@ resource "aws_lb_listener_rule" "routing_https" {
     target_group_arn = aws_lb_target_group.task[each.key].arn
   }
 
+  # condition {
```
Comment (larstobi) [inline modules/fargate/fargate/main.tf:172]: Please delete this comment.

Comment (SadriG91) [reply modules/fargate/fargate/main.tf]: done

**Review by larstobi — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 **/.terraform
 **/*.tfstate*
 crash.log
+**/.terraform.lock.hcl
```
Comment (larstobi) [inline .gitignore:5]: Hmm, I think we should commit these files.

Comment (larstobi) [reply .gitignore:5]: They confirm that the examples work with specific versions.

**Review by larstobi — APPROVED** [review]

**Pull request merged**